### ISSUE TITLE
[BUG] Default Values in Text Fields are not Getting Updated - Supportivekoala | Create Image

### ISSUE THREAD

**Issue by DilanAthukorala** [PipedreamHQ/pipedream#3905]
**Describe the bug**
In supportiveKoala create image action, the user has the ability to update the texts on the selected template before creating the image. The pipdream UI shows the text fields with respective values for each Template Id. But if you switch the template Id then the values of previously selected Template still remains in the text fields without showing values of the switched template id. 

**To Reproduce**
1. Create couple of templates with texts in Supportivekoala App . 
2. In Create Image Action Connect to the SupportiveKoala Account
3. Select a Template Id and observe the text fields with respective values
4. Then switch to another Template Id and observe that text field values are not properly updated. 

**Expected behavior**
Correct values should be filled / updated in the text fields after switching Template Id

**Screenshots**
This is selecting the Template Id initially
![image](https://user-images.githubusercontent.com/102980146/181768740-38881910-b777-42f9-bbeb-155c34d17dca.png)

After switching to a different template with different texts
![image](https://user-images.githubusercontent.com/102980146/181768906-3b75b555-f379-4705-8488-4d8880feb49b.png)

 - Browser - chrome
 - Version [103.0.5060.134 (Official Build) (64-bit)]

**Comment by dannyroosevelt** [comment]
@DilanAthukorala is that true after you click "Refresh Fields" at the bottom of the step configuration?

**Comment by lcaresia** [comment]
This issue is not related with this one https://github.com/PipedreamHQ/pipedream/issues/4538 ?

If yes, does make sense make a merge?

Cc: @dannyroosevelt @DilanAthukorala @Hassan-Abbas-SQA

**Comment by DilanAthukorala** [comment]
> @DilanAthukorala is that true after you click "Refresh Fields" at the bottom of the step configuration?

@dannyroosevelt yes the issue persists even after clicking refresh button on the bottom of the page.

**Comment by DilanAthukorala** [comment]
> This issue is not related with this one #4538 ?
> 
> If yes, does make sense make a merge?
> 
> Cc: @dannyroosevelt @DilanAthukorala @Hassan-Abbas-SQA

@lcaresia This issue is different and not related with 4538
